app/models/activity.rb

```diff
@@ -239,7 +239,14 @@ def associated_fund
   end
 
   def providing_organisation
-    return organisation if third_party_project? && !organisation.is_government?
+    third_party_project? && !organisation.is_government? ? organisation : service_owner
+  end
+
+  def funding_organisation
+    fund? ? nil : service_owner
```

Comment:
I'm never keen on ternary operators, particularly when combined with methods that end with `?`.

My preference would be:

```ruby
def funding_organisation
  return nil if fund?

  service_owner
end
```

But feel free to ignore 😄